### PR TITLE
Added test make target with html test report

### PR DESCRIPTION
**Pull request by afbjorklund**
For best results, install [gotestsum](https://github.com/gotestyourself/gotestsum) and [go-test-report](https://github.com/vakenbolt/go-test-report)

Find the test results in unittest.html and coverage.html

`make test`

Will run `go test -v ./...`, similar to what is in CI (github)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -136,6 +136,21 @@ uninstall:
 	if [ "$$(readlink "$(DEST)/bin/nerdctl")" = "nerdctl.lima" ]; then rm "$(DEST)/bin/nerdctl"; fi
 	if [ "$$(readlink "$(DEST)/bin/apptainer")" = "apptainer.lima" ]; then rm "$(DEST)/bin/apptainer"; fi
 
+.PHONY: test
+test:
```
Comment (AkihiroSuda) [inline Makefile:140]:
```suggestion
test-unit:
```

File: Makefile [hunk 2]
```diff
@@ -136,6 +136,21 @@ uninstall:
...
+	@if command -v gotestsum >/dev/null 2>&1; then \
+		gotestsum --jsonfile unittest.json ./... -coverprofile=coverage.out; \
+	else \
+		go test -v ./... -coverprofile=coverage.out -json >unittest.json; \
+	fi
+	-$(MAKE) --no-print-directory coverage.html unittest.html
+
+unittest.html: unittest.json
```
Comment (AkihiroSuda) [inline Makefile:148]:
```suggestion
test-unit.html: test-unit.json
```

File: Makefile [hunk 3]
```diff
@@ -136,6 +136,21 @@ uninstall:
...
+	go-test-report -o $@ <$<
+
+coverage.html: coverage.out
```
Comment (AkihiroSuda) [inline Makefile:151]:
```suggestion
test-unit-coverage.html: test-unit-coverage.out
```

**Comment by afbjorklund** [comment]
I don't really like "test-unit", so it's back to the drawing board.

**Pull request closed without merging**